web/src/lib/components/shared-components/gallery-viewer/gallery-viewer.svelte

```diff
@@ -32,6 +33,12 @@
 		}
 	}
 
+	const gridLayout = `${
+		dynamicResponsiveGrid
+			? 'grid grid-cols-3 lg:grid-cols-[repeat(auto-fill,minmax(15rem,1fr))]'
```

Comment:
Not necessarily wrong, but why a minimum of 3 columns?

Reply:
Just IMO looks better on mobile screens.